### PR TITLE
lightning: add more retryable err

### LINKED ISSUE #36674: lighting import failed while import on DBAAS
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

1. create 1* TiDB, 3*Tikv 2u8G cluster.
2. try to import 100GB CSV file

### 2. What did you expect to see? (Required)
import success

### 3. What did you see instead (Required)
import fail,  and report Root cause: [Lightning:Restore:ErrRestoreTable]restore table `gotpc_1500`.`order_line` failed: EOF; EOF; EOF.

### 4. What is your TiDB version? (Required)

v6.1.0

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: close #36674

Problem Summary:

### What is changed and how it works?
add retry on `io.EOF` and `coprocessor task terminated due to exceeding the deadline` and `Target table is calculating checksum. Please wait until the checksum is finished and try again.`

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by gozssky — DISMISSED** [review]
Rest LGTM.

File: br/pkg/lightning/backend/local/local.go [hunk 1]
```diff
@@ -1307,6 +1308,19 @@ const (
 	retryIngest
 )
 
+func (local *local) isRetryableTiKVWriteError(err error) bool {
```
[the remark this answers is not in the document]
Comment (D3Hunter) [reply br/pkg/lightning/backend/local/local.go:1311]: this methos is bound to `local` object, it should be a `private static` method, but golang don't have it.

**Comment by D3Hunter** [comment]
add another retryable error ptal @gozssky @buchuitoudegou

**Comment by D3Hunter** [comment]
ci pass 
https://ci.pingcap.net/blue/organizations/jenkins/br_ghpr_unit_and_integration_test/detail/br_ghpr_unit_and_integration_test/12651/pipeline

**Review by gozssky — APPROVED** [review]

**Review by buchuitoudegou — APPROVED** [review]

**Pull request merged**